packages/address/src/tests/AddressFormatter.test.ts

```diff
@@ -0,0 +1,203 @@
+import {fetch} from '@shopify/jest-dom-mocks';
+import {Address, FieldName} from '../types';
+import AddressFormatter from '..';
+import {JapanJa, FranceFr, CanadaEn, CanadaFr} from './fixtures';
+
+const countries = [JapanJa, CanadaFr, FranceFr];
+
+beforeEach(() => {
+  fetch.mock(
```

Comment:
there shouldn't need to be any fetch mocking here if these are unit tests, in which case we could just mock out the loader utility. If you are writing e2e tests, which could also be nice, then do we need to test the loader utility independently at all?